hw_1/02_quadratic_equations.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const quadraticEquation = (a, b, c) => {
+    if(a == 0)
+        return false;
+    let res = {};
+    const D = b * b - 4 * a * c;
+   
+    if(D < 0)
+        return false;
+    res['discriminant'] = D;
+    if(D == 0)
+        res["roots"] = (-b + Math.sqrt(D)) / (2 * a);
```

Comment:
Talking about the result of the function, from.my point of view it should be only the array of roots (1 or 2 doesn't matter). Discriminant is a part of the realisation of function and should stay in function.